test/unit/identifiers.spec.ts

```diff
@@ -845,3 +845,22 @@ test("lua built-in as in constructor assignment", () => {
         export const result = new A("42").error;
     `.expectToMatchJsResult();
 });
+
+test("customName", () => {
```

Comment:
Please give the tests a descriptive name, ie "customName annotation renames function declarations"